test/helpers/kubectl.go

```diff
@@ -460,6 +460,17 @@ func (kub *Kubectl) GetCiliumPodOnNode(namespace string, node string) (string, e
 	return res.Output().String(), nil
 }
 
+// TestConnectivityPodService runs connectivity test from pod to ClusterIP service
```

Comment:
What about 
```
TestConnectivityPodService runs HTTP connectivity test from pod to ClusterIP service
```

Reply:
I left out `HTTP` part because we might want to change the connectivity test itself later. I can put it there, but if we make the connectivity test more sophisticated, we need to reflect this in the comment later